src/app/legacy/config.ts

```diff
@@ -26,6 +26,9 @@ import {
   Matic2xFlexibleLeverageIndexPolygon,
 } from './polygon'
 
+const GitcoinStakedETHIndex = getTokenByChainAndSymbol(1, 'gtcETH')
+const ic21 = getTokenByChainAndSymbol(1, 'ic21')
```

Comment:
Adding directly from tokenlists. Removing lots of obsolete references/assets throughout the app.